File: development/sphinx-console-web/src/main/webapp/sphinx-console/app/model/DbTable.js
Ext.define('sphinx-console.model.DbTable', {
    extend: 'Ext.data.Model',
    autoLoad: true,
    autoSync: true,
    fields: [{name: 'name', type: 'string'}],
    hasMany: [
                {model: 'sphinx-console.model.DbTableColumn', getterName: 'getColumns', setterName: 'setColumns', associationKey: 'columns', associatedName: 'columns' }//, foreignKey: 'id'
             ]

});

File: development/sphinx-console-base/sphinx-console-common-model/src/main/java/ru/skuptsov/sphinx/console/coordinator/model/DBTable.java
package ru.skuptsov.sphinx.console.coordinator.model;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

public class DBTable implements Serializable {
    private String name;
    private String nameWithoutScheme;
    
    private List<DBTableColumn> columns = new ArrayList<DBTableColumn>();

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public List<DBTableColumn> getColumns() {
		return columns;
	}

	public void setColumns(List<DBTableColumn> columns) {
		this.columns = columns;
	}

	public String getNameWithoutScheme() {
		return nameWithoutScheme;
	}

	public void setNameWithoutScheme(String nameWithoutScheme) {
		this.nameWithoutScheme = nameWithoutScheme;
	}
    
    
}
